core/spec/models/spree/product_spec.rb

```diff
@@ -194,6 +194,15 @@ class Extension < Spree::Base
       end
     end
 
+    # Regression test for #8906
+    context 'tags' do
+      let(:tag_list) { %w[tag1 tag2] }
+
+      it 'doesnt raise error when add tags to products' do
+        expect { product.update(tag_list: tag_list) }.not_to raise_error
```

Comment:
`doesnt` -> `doesn't` 
Also - let's change `it` description to something like `doesn't raise an error when adding tags to a product`